pkg/system/names.go

```diff
@@ -16,8 +16,33 @@ limitations under the License.
 
 package system
 
+import (
+	"fmt"
+	"os"
+)
+
 const (
-	// Namespace holds the K8s namespace where our eventing system
-	// components run.
-	Namespace = "knative-eventing"
+	NamespaceEnvKey = "SYSTEM_NAMESPACE"
 )
+
+// Namespace holds the K8s namespace where our eventing system
+// components run.
+func Namespace() string {
```

Comment:
would it make sense to move this to some common location (e.g. to https://github.com/knative/pkg)

looks like relevant for serving, build etc as wll

Reply:
Serving, building and eventing are all belong to knative. I think we can put the three into one namespace. So that we can use helm chart management easily.

Reply:
+1 to @matzew , so that serving, building and eventing will not need to define this function but just import is enough.

Reply:
I agree, it seems like this should be in knative/pkg.